Excelitas-Technologies/docs.excelitasdev.com
# RPW-380: Deploy Rodenstock v1.2.0

# Jira
https://etcprojects.atlassian.net/browse/RPW-380

# Description
* Updated gridsome config.
* Created Rodenstock directory for Releases.
* Added release files.

# Testing
1. Go to Docs Home.
2. Click Rodenstock Docs > [Releases](https://d314ecde.excelitas-docs.pages.dev/rodenstock/releases/)
3. Verify that release documents are added.
![image](https://user-images.githubusercontent.com/1729379/133592020-e96e8ea6-cc18-4399-8c2a-80cefdc9a75f.png)
